app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
They can safely contain a `.` (e.g. see [here](https://rubygems.org/gems/capistrano-db-tasks.senya)). `-` is normally used as ruby namespace separator, which is not strictly the case, but can be used. `diaspora_federation_json_schema` doesn't look very neat.

Also I'm not sure we need `diaspora` there, since we tend to avoid using `diaspora` in protocol related aspects (like we replaced `diaspora_handle` with `author` in entities and so on).